db/migrate/20170924065938_create_restaurant_reviews_metadata.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRestaurantReviewsMetadata < ActiveRecord::Migration[5.1]
+  def change
+    create_table :restaurant_reviews_metadata do |t|
+      t.integer :reviewsCount
```

Comment:
Might be better as `restaurant.reviews_count`

Also see "rails counter_cache"

Reply:
Done,
since the context is a review and not restaurant I think it's more efficient to do it this way, is that OK?